### PR TITLE
Connection/stud 188/connect enrolled school

### PR DESCRIPTION
**Pull request by vithura14**
-After user enters their email and clicks the checkmark, the domain from the email is extracted (everything after the '@' symbol)
-Have a pop up message with the name (of the school in question - obtained from the domain) and its country.

image

### CODE REVIEW

**Comment by nuzurie** [comment]
Is there a reason the merge target branch is develop-test instead of develop?

**Comment by vithura14** [comment]
It was recommended by @stellakimnguyen to have it merged to develop-test.

**Comment by nuzurie** [comment]
In that case, can you please rebase your commits onto develop-test branch? As of now your branch has commits from develop that don't exist in develop-test.

**Comment by vithura14** [comment]
> In that case, can you please rebase your commits onto develop-test branch? As of now your branch has commits from develop that don't exist in develop-test.

Done!

**Comment by stellakimnguyen** [comment]
Can you fix the merge conflicts? Once it's done, let me know! :)

**Comment by vithura14** [comment]
> 

Done!

**Comment by vithura14** [comment]
> Can you fix the merge conflicts? Once it's done, let me know! :)

Done! :)

File: src/components/profile/UserProfile.js [hunk 1]
```diff
@@ -131,6 +135,15 @@ class UserProfile extends Component {
...
     }
+=======
+                error => {
+                    console.log(error)
+                }
+            )         
+    }
+
+
+>>>>>>> c54d8975517cb6d2c205f225dfe0fc11c71f9ac9
```
Comment (stellakimnguyen) [inline src/components/profile/UserProfile.js]: Leaving branch conflict symbols will lead to unevitable crashes, it won't run! To be removed (you'll have to check which one to keep and discard, or potentially keep/discard both)

**Comment by stellakimnguyen** [comment]
According to the commits' list, it seems like there are double commits. Could you make sure there are no duplicated and that the app is running? Ideally we'd remove the duplicated commits so that we don't have commit duplicates once we merge into develop

**Comment by vithura14** [comment]
> According to the commits' list, it seems like there are double commits. Could you make sure there are no duplicated and that the app is running? Ideally we'd remove the duplicated commits so that we don't have commit duplicates once we merge into develop

Yes, my mistake. Did not notice that. App is running fine now!

**Comment by nuzurie** [comment]
Feature redesigned and connected.

**Pull request closed without merging**